adam-core/src/main/scala/org/bdgenomics/adam/converters/DefaultHeaderLines.scala

```diff
@@ -150,7 +150,7 @@ object DefaultHeaderLines {
     VCFHeaderLineType.Float,
     "Read-backed phasing quality")
   lazy val genotypeFilter = new VCFFormatHeaderLine("FT",
-    1,
+    VCFHeaderLineCount.UNBOUNDED,
```

Comment:
Can you keep this split out into a different commit so that we have a single commit per issue? No sweat if you've already squashed.